### DISCUSSION TITLE
Welcome to lua-yottadb Discussions!

### DISCUSSION THREAD

**Discussion by berwynhoyt** [anet-be/lua-yottadb#17] [Announcements]
## 👋 Welcome to lua-yottadb discussion!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ksbhaskar** [comment]
I have lead [YottaDB](https://yottadb.com) since 2017. From 1995 to 2017, I lead the upstream code base.

I don't know whether I am a manager who likes to develop software, or a software developer who likes to manage, or someone completely different. The [YottaDB founders page](https://yottadb.com/about-us/team/) has more information about me.

Send me questions, comments, and concerns about YottaDB. If I don't have an answer, I'll get one for you.

**Comment by AlainDsc** [comment]
I am a software developer, who worked for different companies, writing highly customised stuff, but since 1998, I am working for the Anet Library Automation Team of the Universiteit Antwerpen. The programming languages I use most of the time are Python (since 2004) and M (since 1984). And Lua is coming up..

I have a Mathematics background, and love database designs and generic software.
I have 5 children, 4 stepchildren, and 8 grandchildren so far.
I also play in a Rock cover band (since 1991,as a hobby), drove to Nordcape in a Citroen 2CV in february, and started bee-keeping this year. 

If you have any questions about M, I surely will try to help you.

**Comment by berwynhoyt** [comment]
I am Berwyn Hoyt. I took over development of lua-yottadb from [Mitchell](https://github.com/orbitalquark) to create [version 1.0](https://github.com/anet-be/lua-yottadb/blob/master/docs/changes.md) of lua-yottadb.
